src/Controls/samples/Controls.Sample/Maui.Controls.Sample.csproj

```diff
@@ -53,7 +53,7 @@
 
   <ItemGroup>
     <ProjectReference
-      Include="..\..\..\Controls\src\Core\Controls.SourceGen.csproj"
+      Include="..\..\..\Controls\src\SourceGen\Controls.SourceGen.csproj"
```

Comment:
And indeed this seems like it never existed, even at the time of the PR that introduced it: https://github.com/dotnet/maui/pull/19990/files#diff-af863121b6168780847c8835ede77fb1b01120fa3f9dd5fb48a846128f9c08fdR54-R60